client/state/gravatar-status/selectors.js

```diff
@@ -17,7 +19,5 @@ export function isCurrentUserUploadingGravatar( state ) {
  * @returns {string|boolean} - The temp Gravatar string, or false
  */
 export function getUserTempGravatar( state, userId ) {
-	return (
-		state.currentUser.id === userId && ( state.currentUser.gravatarStatus.tempImage.src ?? false )
-	);
+	return state.currentUser.id === userId && ( state.gravatarStatus.tempImage.src ?? false );
```

Comment:
`!! state.gravatarStatus.tempImage.src` is a bit more stylish 🙂 

Also, if you don't mind, the `tempImage` URL could be directly in `state.gravatarStatus.tempImage`. No need for the extra nested property `.src`.

Then the `state.currentUser.id` could be replaced with the `getCurrentUserId` selector which makes future `currentUser` refactors less error-prone.

Reply:
> `!! state.gravatarStatus.tempImage.src` is a bit more stylish 🙂

Agreed! Addressed in https://github.com/Automattic/wp-calypso/pull/62066/commits/1c9c9934cbcb3f52493af40bcce42c9b3045cf94.

> Also, if you don't mind, the `tempImage` URL could be directly in `state.gravatarStatus.tempImage`. No need for the extra nested property `.src`.

Happy to further simplify as a drive-by, done in https://github.com/Automattic/wp-calypso/pull/62066/commits/26e0c4c5dd9b68aa513d7ec850a715051e42155d

> Then the `state.currentUser.id` could be replaced with the `getCurrentUserId` selector which makes future `currentUser` refactors less error-prone.

Good call, updated in https://github.com/Automattic/wp-calypso/pull/62066/commits/11400e85d816875c6e6af20ce50760273598dcaf.

Reply:
> > !! state.gravatarStatus.tempImage.src is a bit more stylish 🙂

> Agreed! Addressed in https://github.com/Automattic/wp-calypso/commit/1c9c9934cbcb3f52493af40bcce42c9b3045cf94.

Had to revert this, it was breaking the original logic of the selector.